src/goto-synthesizer/dump_loop_contracts.cpp

```diff
@@ -91,16 +91,16 @@ void dump_loop_contracts(
         "loop " + std::to_string(invariant_entry.first.loop_number + 1) +
         " assigns ";
 
-      bool in_fisrt_iter = true;
+      bool in_first_iter = true;
```

Comment:
It's great that this and further typos are being cleaned up, but I'd really appreciate if those could be put in a commit of their own.